Challenge/View/Movies/MoviesList/MoviesListView.swift

```diff
@@ -0,0 +1,259 @@
+//
+//  MoviesListView.swift
+//  Challenge
+//
+//  Created by Lucas Henrique Machado da Silva on 06/05/2018.
+//  Copyright © 2018 Lucas Henrique Machado da Silva. All rights reserved.
+//
+
+import UIKit
+
+class MoviesListView: UIViewController {
+    // MARK: - Outlets
+    @IBOutlet weak var tvMovies: UITableView!
+    @IBOutlet weak var aivLoader: UIActivityIndicatorView!
+    @IBOutlet weak var lMessage: UILabel!
+    @IBOutlet weak var lMore: UILabel!
+    @IBOutlet weak var sbName: UISearchBar!
+    
+    // MARK: - Constants
+    private let refreshControl = UIRefreshControl()
+    
+    // MARK: - Presenter
+    var presenter: MoviesListPresenter?
+    
+    // MARK: - Variables
+    var movies: [Movie]?
```

Comment:
@arctouch-ruannhomem What do you mean be different access modifiers? If you are talking about when calling back the view from the presenter I thought it would be easier to understand if it was separated by what it was doing.